### PR TITLE
Doc: release1.14 + template umbrella issue

### PR DESCRIPTION
**Pull request by churik**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release-issue.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+- [ ] 1. Translations for comms available (marketing, ask in the channel)
+- [ ] 2. Promotional content ready (marketing, ask in channel)
+- [ ] 3. Upload builds for DO store,  https://dev.status.im/ (Jakub)
+
+#### 5) At time of publishing
+- [ ] 1. notify marketing to push comms and blog post (Jonny)
+- [ ] 2. Publish release to app store and playstore
+- [ ] 3. Final push to update prod site https://status.im/,  [instructions](https://github.com/status-im/infra-docs/blob/master/articles/status_release_upload.md) (Jakub - requires credentials to upload) 
+- [ ] 4. Add the link to the post about the release and publish the release: https://github.com/status-im/status-react/releases (Jakub)
+- [ ] 5. update https://github.com/status-im/status-react/blob/develop/RELEASES.md with release notes (make PR based on 1.2. step)
```
Comment (jakubgs) [inline .github/ISSUE_TEMPLATE/release-issue.md]: I still think this should be done way, way earlier, so these notes are available under the release tag and in the release branch.

**Review by jakubgs — APPROVED** [review]
This should be two separate commits. One that updates `RELEASES.md`(which should have been done before we built the final release build), and one for adding the template.

File: .github/ISSUE_TEMPLATE/release-issue.md (hunk 1 above)
Comment (churik) [reply .github/ISSUE_TEMPLATE/release-issue.md]: so just to add commit "update releases.md with n release" on 'Before publishing' stage? Or you can describe the process how to make it available under the release tag (sorry, I'm not familiar with that)? 

If so, I'll update template accordingly

Comment (jakubgs) [reply .github/ISSUE_TEMPLATE/release-issue.md]: It should be done before submission. It needs to be done before the build that will be released is built. Actually, the build for the release should preferrably be done ON the commit that adds the release notes.

**Comment by churik** [comment]
@jakubgs better now, can merge?

File: .github/ISSUE_TEMPLATE/release-issue.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+=============================================================================================
+
+- [ ] Cherry-pick fix(es) to the release branch
+- [ ] Verify fix(es) on new release candidate build
+- [ ] Retest
+- [ ] Keep release build forever (button in Jenkins)
+
+#### 3) Submission
+
+- [ ] 1. Release notes written and added (500 character limit for updates (Android)
```
Comment (jakubgs) [inline .github/ISSUE_TEMPLATE/release-issue.md]: Maybe we should make it clear that we are also talking about `RELEASES.md` in the repo.

Comment (churik) [reply .github/ISSUE_TEMPLATE/release-issue.md]: should be resolved; added om this stage that PR should be ready to merge and on the last stage that it should be merged.

Thank you for thorough review.

**Review by jakubgs — APPROVED** [review]

**Pull request merged**